providers/cstor-operator/test.yml

```diff
@@ -0,0 +1,223 @@
+                  when: lookup('env','CSPC_OPERATOR_IMAGE') | length > 0
+
+                - name: Change the OpenEBS component labels to desired version in Operator yaml
+                  replace:
+                    path: "{{ cspc_operator }}"
+                    regexp: 'openebs.io/version: dev'
+                    replace: "openebs.io/version: {{ cspc_image_tag }}"
+         
+              when: release_version != "master"
+
+          when: "'OK' not in release.msg"
+
+        - block: 
+
+            - name: Change the value for admission server failure policy
+              replace:
+                path: "{{ cspc_operator }}"
+                regexp: 'value: "Fail"'
+                after: '- name: ADMISSION_WEBHOOK_FAILURE_POLICY'
+                replace: 'value: "{{ webhook_policy }}"'
+              when: lookup('env','WEBHOOK_FAILURE_POLICY') != ''
+
+            - name: Applying cspc operator
+              shell: "kubectl  apply -f {{ cspc_operator }}"
+              args:
+                executable: /bin/bash
+
+            - block:
+
+                - name: Applying cspc rbac
+                  shell: "kubectl  apply -f {{ cspc_rbac }}"
+                  args:
+                    executable: /bin/bash
+
+                - name: Applying cspc crd
+                  shell: "kubectl  apply -f {{ cspc_crd }}"
+                  args:
+                    executable: /bin/bash
+              
+              when: "'OK' not in release.msg"
+
+            - name: Checking OpenEBS-CSPC-Operator is running
+              shell: >
+                kubectl get pods -n {{ namespace }}
+                -o jsonpath='{.items[?(@.metadata.labels.name=="cspc-operator")].status.phase}'
+              register: cspc_status
+              until: "'Running' in cspc_status.stdout"
+              delay: 5
+              retries: 120
+
+            - name: Checking OpenEBS-CVC-Operator is running
+              shell: >
+                kubectl get pods -n {{ namespace }}
+                -o jsonpath='{.items[?(@.metadata.labels.name=="cvc-operator")].status.phase}'
+              register: cvc_status
+              until: "'Running' in cvc_status.stdout"
+              delay: 5
+              retries: 120
+
+            - name: Checking OpenEBS-CVC-Operator is running
+              shell: >
+                kubectl get pods -n {{ namespace }}
+                -o jsonpath='{.items[?(@.metadata.labels.app=="cstor-admission-webhook")].status.phase}'
+              register: admission_status
+              until: "'Running' in admission_status.stdout"
+              delay: 5
+              retries: 120                       
+
+          when: lookup('env','ACTION') == "provision"
+
+        - block:
+              - name: Cleaning cspc operator
+                shell: "kubectl  delete -f {{ cspc_operator_link }}"
+                args:
+                  executable: /bin/bash
+                ignore_errors: True

```

Comment:
why do we need ignore_errors  here?

Reply:
If we deleted some of the components already we are getting the error operator  so only used ignore errors @gprasath